app/main.py

```diff
@@ -1,6 +1,21 @@
+import requests
+import os
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+
+    api_key = os.environ.get("API_KEY")
+
+    url = (f"http://api.weatherapi.com/v1/current."
+           f"json?key={api_key}&q=Paris")
+
+    response = requests.get(url)
+
+    data = response.json()
+
+    print(data)
```

Comment:
wrong print

data should be in this format as in description
![image](https://github.com/mate-academy/py-docker-weather-api/assets/104974881/a5c89b8c-157e-408c-8ec3-75e29458b712)